Application Lifecycle Management/instances-api.md

```diff
@@ -1740,226 +2211,644 @@ ElasticBox-Release: 4.0
+[PUT] /services/instances/{instance_id}/reconfigure
+```
+
+#### Example
 
 ```
-Headers:
+[PUT] https://cam.ctl.io/services/instances/i-b1qtkz/reconfigure
+```
+### Request
 
+#### Headers
+```
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
+#### URI Parameters
+Parameter |Type |Description | Req. |
+|-----------|------|------------| ---- |
+| instance_id | string | Instance id | Yes |
 
-```
-Body:
+### Response
+#### Normal Response Codes
+
+- **200** OK
+
+#### Common Error Response Codes
+
+- **409** Operation Conflict
 
+
+#### Response Body
+```
 {
   "id":"i-4g166v",
   "method":"reconfigure"
 }
 ```
 
-### PUT /services/instances/{instance_id}/import
+
+## PUT /services/instances/{instance_id}/import
 
 Retry to import an unregistered instance. when you give its ID.
 
-**Normal Response Codes**
+### URL
 
-* 202
+#### Structure
 
-**Error Response Codes**
+```
+[PUT] /services/instances/{instance_id}/import
+```
 
-* Forbidden (403)
-* Not Found (404)
+#### Example
 
-**Request Parameters**
+```
+[PUT] https://cam.ctl.io/services/instances/i-ndt46z/import
+```
 
-| Parameter | Type |Description |
-|-----------|------|------------|
-| id | string | Instance ID. |
-| method | string | Operation on the instance (import). |
+### Request
 
+#### Headers
 ```
-Headers:
-
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
 
-```
-Body:
+#### URI Parameters
+Parameter |Type |Description | Req. |
+|-----------|------|------------| ---- |
+| instance_id | string | Instance id | Yes |
+
+### Response
+#### Normal Response Codes
+
+- **200** OK
+
+#### Common Error Response Codes
+
+- **403** Forbidden
+- **404** Not Found
+
 
+#### Response Body
+
+```
 {
   "id":"i-eruumb",
   "method":"import"
 }
 ```
 
-### PUT /services/instances/{instance_id}/cancel_import
+## PUT /services/instances/{instance_id}/cancel_import
 
 Cancel a failed import of an unregistered instance when you give its ID.
 
-**Normal Response Codes**
+### URL
 
-* 202
+#### Structure
 
-**Error Response Codes**
+```
+[PUT] /services/instances/{instance_id}/cancel_import
+```
 
-* Forbidden (403)
-* Not Found (404)
+#### Example
 
-**Request Parameters**
+```
+[PUT] https://cam.ctl.io/services/instances/i-ndt46z/cancel_import
+```
 
-| Parameter | Type |Description |
-|-----------|------|------------|
-| id | string | Instance ID. |
-| method | string | Operation on the instance (cancel_import). |
+### Request
 
+#### Headers
 ```
-Headers:
-
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
 
-```
-Body:
+#### URI Parameters
+| Parameter | Type |Description |Req.|
+|-----------|------|------------|----|
+| instance_id | string | Instance id | Yes |
+### Response
+#### Normal Response Codes
+
+- **200** OK
+
+#### Common Error Response Codes
 
+- **403** Forbidden
+- **404** Not Found
+
+
+#### Response Body
+
+```
```

Comment:
Change the last line in this document
old: * Windows: RDP into the instance to locate the log at ProgramDataElasticBoxLogselasticbox-agent.log
new: * Windows: RDP into the instance to locate the log at \ProgramData\ElasticBox\Logs\elasticbox-agent.log